src/js/samples/components/Files/SampleReads.tsx

```diff
@@ -14,10 +14,16 @@ const SampleReadsTitle = styled.h2`
     }
 `;
 
-function SampleReads({ reads, prefix }) {
-    const fileComponents = map(reads, (file, index) => (
-        <ReadItem key={file.name} {...file} prefix={prefix} suffix={index + 1} />
-    ));
+type SampleReadsProps = {
+    /** A list of reads used to create the sample */
+    reads: Read[];
+};
+
+/**
+ * Displays a list of reads used to create the sample
+ */
+export default function SampleReads({ reads }: SampleReadsProps) {
+    const fileComponents = map(reads, file => <ReadItem key={file.name} {...file} />);
```

Comment:
```suggestion
    const fileComponents = map(reads, file => <ReadItem key={file.name} name={file.name} />);
```
I generally avoid spread syntax as it can make what is getting passed a bit less obvious. I tend to either pass only the needed props or pass the whole reference